src/Draco.Editor.Web/app/ts/app.ts

```diff
@@ -5,12 +5,14 @@ import { Registry } from 'monaco-textmate';
 import { wireTmGrammars } from 'monaco-editor-textmate'; // Library that allow running Textmates grammar in monaco.
 import grammarDefinition from '../../../Draco.SyntaxHighlighting/draco.tmLanguage.json';
 import { deflateRaw, inflateRaw } from 'pako';
+import { stdout } from 'process';
 
 // This file is run on page load.
 // This run before blazor load, and will tell blazor to start.
 
-const worker = new Worker('worker.js'); // first thing: we start the worker so it loads in parallel.
-
+const compilerWorker = new Worker('worker.js'); // first thing: we start the worker so it loads in parallel.
+let runtimeWorker : Worker | undefined;
```

Comment:
Isn't the convention `name: type` and not `name : type`?

Reply:
Yeah, autoformat is breaking the linter convention, was too lazy to spend time fixing it so no autoformating ^^'.